datadog_lambda/wrapper.py

```diff
@@ -204,8 +206,10 @@ def __init__(self, func):
             patch_all()
 
             logger.debug("datadog_lambda_wrapper initialized")
-        except Exception:
-            traceback.print_exc()
+        except Exception as e:
+            logger.error(
+                "Error %s. Traceback: %s", e, traceback.format_exc().replace("\n", "\r")
```

Comment:
One minor nit, we might consider splitting this out into a small util method, as we've got it in a few places and there doesn't seem to be something we'd change depending on input. Maybe the message? Even that can be a param.

Reply:
been thinking about this. will do.